backend/app/utils/jwt_utils.py

```
@@ -11,28 +13,71 @@
+            detail=f"{token_type.title()} token has expired"
+        )
     except jwt.InvalidTokenError:
-        raise HTTPException(status_code=401, detail="Invalid token")
+        raise HTTPException(
+            status_code=401,
+            detail=f"Invalid {token_type} token"
+        )
+
+def verify_and_get_refresh_token(token: str, db: Session) -> RefreshToken:
+    # Verify JWT signature and expiration
+    payload = verify_token(token, "refresh")
```

Ruff (0.11.9)

Local variable `payload` is assigned to but never used

Remove assignment to unused variable `payload`

(F841)